crates/wash/tests/start_wasmcloud.rs

```diff
@@ -68,10 +68,13 @@ async fn can_download_and_start_wasmcloud() -> anyhow::Result<()> {
     .await
     .expect("Unable to start nats process");
 
-    let wasmcloud_binary =
-        ensure_wasmcloud(&Version::parse(WASMCLOUD_VERSION).unwrap(), &install_dir)
-            .await
-            .expect("Unable to ensure wasmcloud");
+    let wasmcloud_binary = ensure_wasmcloud(
```

Comment:
Could you add a test that spins up a simple server to return the binary? It can probably call the lower level download from GH stuff, but just so there is an integration test that this works